src/Utils/Utils.php

```diff
@@ -262,19 +285,50 @@ public static function fetchNavigationCategoryFromSalesChannel(
         return $navigationCategory;
     }
 
-    /**
-     * Builds the category path by removing the path of the parent (root) category of the sales channel.
-     * Since Findologic does not care about any root categories, we need to get the difference between the
-     * normal category path and the root category.
-     *
-     * @return string[]
-     */
-    private static function getCategoryBreadcrumb(array $categoryBreadcrumb, CategoryEntity $rootCategory): array
+    public static function parseFindologicFiltersForShopware(FiltersExtension $filtersExtension): array
```

Comment:
This method shouldn't be in here. Please move it somewhere else. I think it fits best in `FilterHandler`.